chpl/chpl-service/src/main/java/gov/healthit/chpl/dto/TestFunctionalityCriteriaMapDTO.java

```diff
@@ -0,0 +1,91 @@
+package gov.healthit.chpl.dto;
+
+import java.util.Date;
+
+import gov.healthit.chpl.entity.TestFunctionalityCriteriaMapEntity;
+
```

Comment:
JavaDoc? I know checkstyle would have complained about this

Reply:
Added